File: QViz.NET/Periscope.QViz/JSON/TestSuite.cs
namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Test Suite Details
	/// </summary>
	public class TestSuite
	{
		/// <summary>
		/// Identifier of the Test Suite
		/// </summary>
		public string testsuiteId;
		
		/// <summary>
		/// Name of the Test Suite
		/// </summary>
		public string testsuiteName;
		
		/// <summary>
		/// Is this Test Suite Active?
		/// </summary>
		public bool isActive;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public TestSuite()
		{
			this.testsuiteId = "";
			this.testsuiteName = "";
			this.isActive = false;
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/TestSuite.java
package com.periscope.qviz.json;

/**
 * JSON Model for Test Suite Details
 */
public class TestSuite {

	/**
	 * Identifier of the Test Suite
	 */
	public String testsuiteId;

	/**
	 * Name of the Test Suite
	 */
	public String testsuiteName;

	/**
	 * Is this Test Suite Active?
	 */
	public Boolean isActive;

	/**
	 * Default Constructor
	 */
	public TestSuite() {
		this.testsuiteId = "";
		this.testsuiteName = "";
		this.isActive = false;
	}

}
